storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -31,6 +32,7 @@
 
 (defmulti download-storm-code cluster-mode)
 (defmulti launch-worker (fn [supervisor & _] (cluster-mode (:conf supervisor))))
+(defmulti mk-bt-tracker cluster-mode)
```

Comment:
Can we rename this? it is not bit torrent so bt-tracker is really a confusing name.